docs/ems/recruitment_privacy.md

```diff
@@ -1,42 +1,79 @@
+For the purpose of the General Data Protection Regulation (“GDPR”) and the Data Protection Act 2018 ("DPA18") the Data Controller for your data is Element, a trading name of New Vector Ltd.
 
 
-## Your personal information
+## **Your personal information**
 
-### Information we collect from you
+
+### **Information we collect from you and the reasons**
 
 We collect and process some or all of the following types of information from you:
 
-1. Information that you provide when you apply for a role. This includes information provided through an online job site, via email, in person at interviews and/or by any other method.
-1. In particular, we  process personal details such as name, email address, address, date of birth, qualifications, experience, information relating to your employment history, skills and experience that you provide to us.
-1. If you contact us, we  may keep a record of that correspondence.
-1. Details of all actions that you carry out through the process.
+1. Information that you provide when you apply for a role. This includes information provided through an online job site, to a recruiter, via email, in person at interviews and/or by any other method.
+
+2. In particular, we may process personal details such as name, email address, address, date of birth, qualifications, experience, information relating to your employment history, skills and experience that you provide to us. This information is used for correspondence in relation to the recruitment process and to assess suitability for the role.
+
+3. If you contact us, we may keep a record of that correspondence.
+
+4. Details of all actions that you carry out through the process.
+
+5. We may need to process your health information in order to make reasonable adjustments to the recruitment selection process.
+
+6. We may collect special categories data for equal opportunities, diversity and inclusion purposes. This includes information about health, ethnic origin, sexual orientation or religious beliefs.
+
+We collect and store your information using Workable, as a data processor. You can review their privacy policy[ here](https://www.workable.com/privacy).
+
+If your application is unsuccessful we will hold your data (in our systems) for 12 months after the recruitment process concludes. After this point it will be deleted.
+
+If you are successful in your application, data collected will be transferred to our internal HR systems and this will be used for onboarding and during your employment in accordance with our employee privacy policy.
+
+
+### **Information we collect from other sources**
+
+On application we may follow links provided by you directly or indirectly to publicly available information that you have published on the internet – this may include sources such as Github and other social media profiles, such as LinkedIn.
```

Comment:
```suggestion
On application we may follow links provided by you directly or indirectly to publicly available information that you have published on the internet – this may include sources such as GitHub and other social media profiles, such as LinkedIn.
```